pimd/pim_igmpv3.c

```diff
@@ -1827,6 +1828,68 @@ void igmp_v3_recv_query(struct igmp_sock *igmp, const char *from_str,
 	} /* s_flag is clear: timer updates */
 }
 
+static bool igmp_pkt_grp_addr_ok(struct interface *ifp, const char *from_str,
+				 struct in_addr grp, int rec_type)
+{
+	struct prefix lncb;
+	struct prefix g;
+	struct pim_interface *pim_ifp;
+
+	pim_ifp = ifp->info;
+
+	/* determine filtering status for group */
+	if (pim_is_group_filtered(pim_ifp, &grp)) {
+		if (PIM_DEBUG_IGMP_PACKETS)
+			zlog_debug(
+				"Filtering IGMPv3 group record %s from %s on %s per prefix-list %s",
+				inet_ntoa(grp), from_str, ifp->name,
+				pim_ifp->boundary_oil_plist);
+		return false;
+	}
+
+	/*
+	 * If we receive a igmp report with the group in 224.0.0.0/24
+	 * then we should ignore it
+	 */
+
+	lncb.family = AF_INET;
+	lncb.u.prefix4.s_addr = 0x000000E0;
```

Comment:
please use str2prefix to avoid endianness issues in the future.  hard coding something like this will be a fun debug.